### PR TITLE
My first pull Request!!

### PR DESCRIPTION
**Pull request by girma336**
this is my first pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,21 +35,7 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
```
Comment (franclobo) [inline README.md:36]: - [ ] Please 🙏🏽 replace it with your name. It is your README file.

File: README.md [hunk 2]
```diff
@@ -72,4 +51,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (franclobo) [inline README.md:52]: - [ ] Kindly make sure that all your links work as expected, we can replace the name of the file LICENSE with MIT.md like this 👉🏽 `[MIT](./MIT.md)`.

File: MIT.md [hunk 3]
```diff
@@ -1,10 +1,3 @@
...
+## Copyright 2022, [Girma Tarekegn]
 
-###### Please delete this line and the next one
-###### APP TYPE can be a webpage/website, a web app, a software and so on
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
-
-THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].
```
Comment (franclobo) [inline MIT.md:10]: - [ ] Please 🙏🏽 it is not necessary to modify the LICENSE file. We can create it again following these [steps](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (franclobo) [inline .gitignore:1]: - [ ] Please 🙏🏽 add the node_module folder to the .gitignore file.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @girma336, 

Good job so far!
You are doing it well 👍🏽 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Decriptive commits ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly replace the README sections with your own information and delete those that your are not using yet.

image

- [ ] Please 🙏🏽 delete the [app_screenshot.png](https://github.com/girma336/Hello-World/blob/new-branch/app_screenshot.png) file because you are not using it in  this project.
- [ ] We could add Microverse in the Acknowledgments section 👇🏽 
image

- [x] Kindly add the linters 👇🏽 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (girma336) [reply README.md:15]: Ok thanks. I have done

File: README.md (hunk 2 above)
Comment (girma336) [reply README.md]: done

**Pull request merged**